provider.tf

```diff
@@ -0,0 +1,12 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 3.0"
+    }
+  }
+}
+# Configure the AWS Provider
+provider "aws" {
+  region = "us-west-2"
+}
```

Comment:
Please add a newline to the end of the file

Reply:
fixed